documents/coredocument.go

```diff
@@ -651,14 +651,24 @@ func (cd *CoreDocument) Timestamp() (time.Time, error) {
 }
 
 // AddAttributes adds a custom attribute to the model with the given value. If an attribute with the given name already exists, it's updated.
-func (cd *CoreDocument) AddAttributes(attrs ...Attribute) (*CoreDocument, error) {
+func (cd *CoreDocument) AddAttributes(ca *CollaboratorsAccess, prepareNewVersion bool, attrs ...Attribute) (*CoreDocument, error) {
```

Comment:
Why do we need the prepareNewVersion flag now? And not only relaying on the `ca` value.

Reply:
Discussed in person